apps/backoffice-v2/src/lib/zod/schemas/end-users.ts

```diff
@@ -14,7 +14,7 @@ export const EndUsersListSchema = z
       email: z.string().nullable().default(''),
       phone: z.string().nullable().default(''),
       approvalState: z.enum(States).default(State.PROCESSING),
-      assignedTo: z.string().default(''),
+      assignedTo: z.string().nullable().default(null),
```

Comment:
The "empty state" of `assignedTo` should be null instead of an empty string. When it's null it's obvious that it's unassigned.